pylint/test/input/func_e0303.py

```diff
@@ -0,0 +1,44 @@
+"""Check bad value returned by __len__ """
```

Comment:
We don't write these kind of tests anymore, instead our current functional tests are residing in test/functional. The format is simple, you just have to annotate with a comment the lines that are going to emit the problem. Take a look in that directory, there are plenty of examples.